mozilla/bedrock
# Fix Bug 1393622 - Updates to MozSpace contact pages (Aug 2017)

## Description

Remove the Auckland and Tokyo spaces as those offices no longer exist. Should be merged ASAP.

## Issue / Bugzilla link

[Bug 1393622](https://bugzilla.mozilla.org/show_bug.cgi?id=1393622)

## Testing

* Make sure both `/contact/spaces/auckland/` and `/contact/spaces/tokyo/` are redirected to `/contact/spaces/`
* Make sure the global map on `/contact/spaces/` no longer shows the Auckland and Tokyo spaces
